src/kirby/components/chart/chart.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, OnInit, Input, OnChanges, ElementRef, ViewChild } from '@angular/core';
+import { Options } from 'highcharts';
+import { ChartHelper } from './chart-helper';
+import { DonutOptions } from './options/donut';
+import { ChartTypes } from './chart-types';
+
+@Component({
+  selector: 'kirby-chart',
+  templateUrl: './chart.component.html',
+  styleUrls: ['./chart.component.scss']
+})
+
+export class ChartComponent implements OnInit, OnChanges {
+  @Input() data = [];
+  @Input() height = 300;
+  @Input() type: ChartTypes = ChartTypes.PIE;
+  @Input() description = '';
+  @Input() dataLabelsEnabled = true;
+
+  options: Options = {};
+  chartHelper: ChartHelper;
+
+  constructor() {
+    this.chartHelper = new ChartHelper();
+  }
+
+  @ViewChild('chartContainer') chartContainer: ElementRef;
+
+  ngOnInit() {
+    this.setupChartType();
+    this.updateProperties();
+    this.chartHelper.init(this.options, this.chartContainer);
+  }
+
+  ngOnChanges() {
+    this.updateProperties();
+    this.chartHelper.onChanges(this.options);
```

Comment:
Det er vel kun komponentens metode, der er en eventhandler og dermed skal hedde `onSomething`?
```suggestion
    this.chartHelper.updateChart(this.options);
```